frontend/react/src/components/fields/SynthesizedTable.js

```diff
@@ -35,21 +35,34 @@ const SynthesizedTable = ({ question, rows }) => {
 };
 SynthesizedTable.propTypes = {
   question: PropTypes.object.isRequired,
+  headers: PropTypes.array.isRequired,
   rows: PropTypes.array.isRequired,
 };
 
 const mapStateToProps = (state, { question }) => {
+  const headers = question.fieldset_info.headers.map((header) => {
+    return header.contents === "" ? { contents: "Type" } : header;
+  });
+
   const rows = question.fieldset_info.rows.map((row) =>
     row.map((cell) => {
       const value = synthesizeValue(cell, state);
 
-      return typeof value.contents === "number" && Number.isNaN(value.contents)
-        ? { contents: "Not Available" }
-        : value;
+      if (typeof value.contents === "number" && Number.isNaN(value.contents)) {
+        return { contents: "Not Available" };
+      } else if (!value.contents) {
+        return { contents: "Not Answered" };
+      } else if (Array.isArray(value.contents)) {
+        return value.contents === ""
+          ? { contents: "Not Answered" }
+          : { contents: value.contents[0] };
+      } else {
+        return value;
+      }
```

Comment:
Don't need this else here since if it returns higher up it won't reach it.
```suggestion
        return value;
```